internal/metrics/feature_test.go

```diff
@@ -210,7 +218,11 @@ func TestHandleRequestInvalidFrequency(t *testing.T) {
 	msg := things.NewMessage(model.NewNamespacedIDFrom(testThingID)).
 		Feature(featureID).
 		WithPayload("{'frequency':'invalid_duration'}")
-	metrics.client.Send(msg.Inbox(testOperationRequest).Envelope(protocol.WithCorrelationID("request_invalid_duration")))
+
+	metrics.client.Send(msg.Inbox(testOperationRequest).
+		Envelope(protocol.WithResponseRequired(true),
+			protocol.WithCorrelationID("request_invalid_duration")))
```

Comment:
Maybe it is a good idea to also add a test that checks that no response is received when response required is not set.